sw/vendor/riscv_compliance/riscv-target/opentitan/device/rv32imc/wrap.c

```diff
@@ -0,0 +1,34 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include <string.h>
+
+#include "common.h"
+#include "uart.h"
+
+#define SIM_TERM_ADDR 0x10008000
+
+void dump_signature(void) {
+  extern uint32_t begin_signature[];
+  extern uint32_t end_signature[];
+
+  uint32_t size = end_signature - begin_signature;
+  uart_init(UART_BAUD_RATE);
+  // uart_send_str("PASS!\r\n");
+  for (uint32_t i = 0; i < size; i++) {
```

Comment:
++i, not i++.

Reply:
updated, but does it make a difference here..?